tests/test_cfgfast.py

```diff
@@ -556,10 +556,12 @@ def test_function_leading_blocks_merging():
                                 )
 
     nose.tools.assert_in(0x8000799, cfg.kb.functions, "Function 0x8000799 does not exist.")
+    nose.tools.assert_not_in(0x800079b, cfg.kb.functions, "Function 0x8000799 does not exist.")
```

Comment:
Maybe this assert_msg is `"Function 0x800079b does exist."` ?

Reply:
Fixed. Thank you!